File: Backend/WsParqueadero/WsParqueadero/Models/ResAutenticar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WsParqueadero.Models
{
    public class ResAutenticar
    {
        public String Codigo { get; set; }
        public String Mensaje { get; set; }
        public Pago pago { get; set; }
        public Usuario usuario { get; set; }
    }
}

File: Frontend/WebParqueadero/src/app/Modelo/ResAutenticar.ts
import { Usuario } from './usuario';
import { Pago } from './Pago';

// export class ResAutenticar {
//     // tslint:disable-next-line:ban-types
//     Codigo: String;
//     // tslint:disable-next-line:ban-types
//     Mensaje: String;
//     pago: Pago;
// }


export interface ResAutenticar {
  Codigo: string;
  Mensaje: string;
  pago: Pago;
  Usuario: Usuario;
}

